core/src/main/java/io/aiven/klaw/service/TopicControllerService.java

```diff
@@ -1287,12 +1112,12 @@ private List<TopicInfo> getTopicInfoList(
     return topicsListMap;
   }
 
-  private String getUserName() {
+  public String getUserName() {
     return mailService.getUserName(
         SecurityContextHolder.getContext().getAuthentication().getPrincipal());
```

Comment:
```suggestion
    return mailService.getUserName(getPrincipal());
```
to avoid duplication

Reply:
agree, done.